change-notes/1.22/analysis-cpp.md

```diff
@@ -11,6 +11,7 @@
 
 | **Query**                  | **Expected impact**    | **Change**                                                       |
 |----------------------------|------------------------|------------------------------------------------------------------|
+| Continue statement that does not continue (`cpp/continue-in-false-loop`) | Fewer false positive results | Analysis is now restricted to `do`-`while` loops. |
```

Comment:
The change note needs to say something about how this query is now displayed by default on LGTM even though it wasn't even run on LGTM before. @semmledocs-ac, does that mean we should move the change note entry to the "New queries" section? It's not technically a new query since we already shipped it to paying customers.

Reply:
If customers were aware of this query previously we should keep it in the "Changes to existing queries" section, but state that it will now be run. If not then we can treat it as a new query.